.github/workflows/nestml-build.yml

```diff
@@ -123,7 +123,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        nest_branch: ["v2.20.2", "v3.0", "v3.7", "master"]
```

Comment:
Should the v3.8_rc1 changes be in this branch/PR?

Reply:
Merged it by mistake. Removed it now.